internal/pkg/daemon/enricher/container.go

```diff
@@ -124,6 +116,10 @@ func (e *Enricher) getContainerInfo(
 						recordProfile: recordProfile,
 					}
 
+					if e.labelProblematicContainers {
+						go e.impl.LabelProblematicPod(e.clientset, info.containerName, info.podName, info.namespace, e.logger)
```

Comment:
Do you think it would decrease the noise to only label pods that are not being recorded? Or do you think that using this feature is more or less exclusive with recording? (Either way, nothing that should block this patch, just thinking out loud).

For future I guess we could have a simple namespace selector or even a list of NSs in case we want to restrict what pods are labeled this way (here I'm thinking about that use-case with watching just-recorded policies for additional AVCs...)

Reply:
So, this currently tails audit logs, so it will label ANY pod running on a given node. We could start limiting that with a label selector, but that would only take effect until after the pod has been parsed from the node itself.